### ISSUE TITLE
Automatically open the failed diffs

### ISSUE THREAD

**Issue by nicrodgers** [B2F/Succss#6]
Is it possible to get succss to automatically open the failed images after completing a 'check' run? 

On MacOS for example, we'd need to run "open [path/to/diff]" for each failure. 

It'd be really helpful, as sometimes when the test run shows a lot of failures, it's quite fiddly having to scrollback through the output and copy/paste the path to each image, so this would really help to save time.

**Comment by SLyHuy** [comment]
No, but you can check the customize by me. https://github.com/SLyHuy/Succss
It have visual report on browser and json report file too.

**Comment by B2F** [comment]
You can also try the 2.x-dev branch on the Succss github project, it has a spa reporting page made with angularjs.

**Comment by FDiskas** [comment]
And how to use this? I installed dev branch and after all tests pass I got no reports or don't know how to access them
